club/theme.json

```diff
@@ -129,26 +129,81 @@
 					]
 				}
 			],
+            "fluid": true,
             "fontSizes": [
                 {
-                    "name": "Small",
                     "size": "1rem",
-                    "slug": "small"
+                    "fluid": {
+                        "min": "0.875rem",
+                        "max": "1rem"
+                    },
+                    "slug": "small",
+                    "name": "Small"
+                },
+                {
+                    "size": "1.25rem",
+                    "slug": "default",
+                    "name": "Default"
                 },
                 {
-                    "name": "Medium",
                     "size": "1.5rem",
-                    "slug": "medium"
+                    "slug": "medium",
+                    "name": "Medium"
+                },
+                {
+                    "size": "1.75rem",
+                    "slug": "large",
+                    "name": "Large"
+                },
+                {
+                    "size": "2.5rem",
+                    "slug": "x-large",
+                    "name": "Extra Large"
+                },
+                {
+                    "size": "3.75rem",
+                    "fluid": {
+                        "min": "3.75rem",
+                        "max": "8.125rem"
+                    },
+                    "slug": "header-one",
+                    "name": "Header 1"
+                },
+                {
+                    "size": "2.5rem",
+                    "fluid": {
+                        "min": "2.5rem",
+                        "max": "6.875rem"
+                    },
+                    "slug": "header-two",
+                    "name": "Header 2"
+                },
+                {
+                    "size": "1.875rem",
+                    "fluid": {
+                        "min": "1.875rem",
+                        "max": "2.5rem"
+                    },
+                    "slug": "header-three",
+                    "name": "Header 3"
                 },
                 {
-                    "name": "Large",
                     "size": "1.75rem",
-                    "slug": "large"
+                    "fluid": {
+                        "min": "1.75rem",
+                        "max": "2rem"
+                    },
+                    "slug": "header-four",
+                    "name": "Header 4"
                 },
                 {
-                    "name": "Extra Large",
-                    "size": "2rem",
-                    "slug": "x-large"
+                    "size": "1.25rem",
+                    "fluid": {
+                        "min": "1.25rem",
+                        "max": "2.5rem"
+                    },
+                    "slug": "default-title",
+                    "name": "Default Title"
```

Comment:
Should all these font settings be added in this PR, or will they be covered by https://github.com/Automattic/themes/issues/6213?

Reply:
Nope, the PR is a little bit dirty with other changes. For simplicity, I will close this PR and open a new one.